src/graphql/error-map.ts

```diff
@@ -162,6 +162,10 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
       message = "Unsupported phone carrier for rewards."
       return new ValidationInternalError({ message, logger: baseLogger })
 
+    case "InvalidIpMetadataForRewardError":
+      message = "Unsupported IP address for rewards."
```

Comment:
I'm wondering if `Invalid` is not better than `Unsupported`? or maybe some other word could be used. `Unsupported` feels off to me in term of vocabulary.

Reply:
I was mirroring the verbiage used in `InvalidPhoneMetadataForRewardError`, which returns a message of `Unsupported phone carrier for rewards.`. 

IMHO, `Unsupported` is better than `Invalid` for both the phone metadata and IP metadata error. To me, `Invalid` would mean that the IP address / phone number doesn't exist or is incorrectly formatted, while `Unsupported` means that the application doesn't support rewards for that (syntactically correct) IP address / phone number.

Of course, just my opinion and I'm known to be terrible at naming things :). 

I'd be happy to change it, but would ask if you want to change the verbiage for both IP metadata & phone metadata, or just the IP metadata.

Reply:
I think a word like `blocked` might also work. `Your phone carrier is blocked from receiving rewards`, `Your IP is blocked from receiving rewards`

Reply:
I like the text that @bodymindarts proposes. That seems to be the most descriptive way of telling the user this. Change pushed.